services/app-api/src/postgres/contractAndRates/findContractWithHistory.ts

```diff
@@ -46,10 +46,16 @@ async function findContractWithHistory(
             },
         })
 
+        if (!contract) {
+            const err = `PRISMA ERROR: Cannot find contract with id: ${contractID}`
+            console.error(err)
+            return new NotFoundError(err)
+        }
+
         return parseContractWithHistory(contract)
     } catch (err) {
         console.error('PRISMA ERROR', err)
-        return convertPrismaErrorToStoreError(err)
```

Comment:
Are there any other store errors we should be returning instead of just the raw prisma error? I'm not sure we handle any of them so we can leave it probably.

Reply:
I could not think of any others. I'm also good with leaving it and if others creep up, we can add them. I think the main factor for making a custom error is if we are checking for specific errors in the resolvers, like `NotFoundError`.